docs/reference/mount-write-mode-limitations.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Mount Write Mode Limitations
+description: Limitations of the mount write mode
+grand_parent: Features
+parent: Mount
+---
+
+# Mount Write Mode Limitations
+
+## OS Limitations
+- Mount write mode is currently supported only on macOS.
+
+## Everest Partial-Read Mode
+- Mount write mode is not supported with Everest partial-read mode.
+
+## Functionality Limitations
+- Newly created empty directories will not reflect as directory marker in lakeFS.
+
+## File System Limitations
```

Comment:
Please see in this [design section](https://github.com/treeverse/dev/blob/main/designs/open/everest-mount/mount-write.md#initial-release-limitations) 
Split the limitations to not-supported, partially supported and behavior modified. 
Then explain the things there.